homeassistant/components/fritz/common.py

```diff
@@ -175,15 +180,16 @@ def setup(self) -> None:
 
         self._update_available, self._latest_firmware = self._update_device_info()
 
-    async def async_start(self, options: MappingProxyType[str, Any]) -> None:
-        """Start FritzHosts connection."""
-        self.fritz_hosts = FritzHosts(fc=self.connection)
-        self._options = options
-        await self.hass.async_add_executor_job(self.scan_devices)
-
-        self._cancel_scan = async_track_time_interval(
-            self.hass, self.scan_devices, timedelta(seconds=TRACKER_SCAN_INTERVAL)
-        )
+    @callback
+    async def _async_update_data(self) -> None:
+        """Update FritzboxTools data."""
+        try:
+            self.fritz_hosts = FritzHosts(fc=self.connection)
+            await self.async_scan_devices()
+        except FritzSecurityError as ex:
+            raise update_coordinator.UpdateFailed("Error fetching data") from ex
+        except FritzConnectionException as ex:
+            raise update_coordinator.UpdateFailed("Error fetching data") from ex
```

Comment:
```suggestion
        except (FritzSecurityError, FritzConnectionException) as ex:
            raise update_coordinator.UpdateFailed from ex
```
- or was there a reason to catch them separately?
- adding a message is not needed, since already done in parent
https://github.com/home-assistant/core/blob/3471a44f923c0f729b7f6eefb73ba6dd8539297e/homeassistant/helpers/update_coordinator.py#L215-L220

Reply:
Done